Update border UI rules

jordanjones243 opened the pull request:
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #228 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)


## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

jordanjones243 committed:
fix(border): update UI border rules #228

blackfalcon commented:
@jordanjones243 can you please add BEFORE and AFTER screenshots 

@leeejune here is a demo page
https://alaskaairlines-auro-dropdown-pr-229.surge.sh/
https://alaskaairlines-auro-dropdown-pr-229.surge.sh/api.html

blackfalcon committed:
build: update build scripts

This commit updates the build scripts to support this Discussion
https://github.com/orgs/AlaskaAirlines/discussions/523

This commit also updates all dependencies with new versions.

Changes to be committed:
modified:   .github/workflows/testPublish.ym
modified:   package-lock.json
modified:   package.json

blackfalcon commented:
> @jordanjones243 can you please add BEFORE and AFTER screenshots
> 
> @leeejune here is a demo page https://alaskaairlines-auro-dropdown-pr-229.surge.sh/ https://alaskaairlines-auro-dropdown-pr-229.surge.sh/api.html

@leeejune the latest update that @jordanjones243 has been updated in that Surge link.

jordanjones243 commented:
This PR is no longer needed as the ticket has been closed
